internal/testing/runchecks.sh

```diff
@@ -106,6 +107,13 @@ else
 fi;
 
 
+echo
+echo "Ensuring that go mod tidy has been run for the root module..."
```

Comment:
Sidenote: it's annoying that runcheks does $THINGS for the root module and then repeats them for submodules. The inherent difference seems to be that we're running some extra stuff for root (like coverage). Perhaps there's a cleaner way to do that all in one loop. Just thinking aloud for the future

Reply:
Ack.

Reply:
We could probably move the "loop over modules" stuff to the top, and do `go test` + `wire` etc. in that loop, with a conditional for coverage. Although, we probably want coverage for more than the root module... e.g., we'll want it for the driver submodules, and for `gocdk`.

Reply:
Filed https://github.com/google/go-cloud/issues/2117.